docs/how-to/troubleshooting.md

```diff
@@ -125,21 +125,21 @@ HOLOVIZ_MCP_PORT=8001 HOLOVIZ_MCP_TRANSPORT=http uvx holoviz-mcp
 
 ### Documentation Index Creation Fails
 
-**Problem**: `holoviz-mcp-update` fails or times out.
+**Problem**: `holoviz-mcp update` fails or times out.
 
 **Solution**:
 
 1. Check internet connection
 
 2. Try again with debug logging:
    ```bash
-   HOLOVIZ_MCP_LOG_LEVEL=DEBUG uvx --from holoviz-mcp holoviz-mcp-update
+   HOLOVIZ_MCP_LOG_LEVEL=DEBUG holoviz-mcp update
```

Comment:
I do believe this should be `HOLOVIZ_MCP_LOG_LEVEL=DEBUG uvx holoviz-mcp update`